operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -193,25 +197,46 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
         
         List<String> resourceTypesOverride = getOverridenIncludedResourceTypes(parameters, defaultResourceTypes);
         List<String> resourceTypes = resourceTypesOverride.isEmpty() ? defaultResourceTypes : resourceTypesOverride;
-
+        List<String> resourceIds = new ArrayList<String>();
+        
         int totalResourceCount = 0;
         for (String compartmentType : resourceTypes) {
             MultivaluedMap<String, String> searchParameters = queryParameters;
             if (startOrEndProvided  && !SUPPORT_CLINICAL_DATE_QUERY.contains(compartmentType)) {
                 LOG.finest("The request specified a '" + START_QUERY_PARAMETER + "' and/or '" + END_QUERY_PARAMETER + "' query parameter. They are not valid for resource type '" + compartmentType + "', so will be ignored.");
                 searchParameters = queryParametersWithoutDates;
             }
-
+            
```

Comment:
```suggestion

```

Reply:
we usually configure eclipse to remove trailing whitespace on all lines...that should reduce some of the formatting-related churn on these files.  I screenshotted the prop at https://github.com/IBM/FHIR/wiki/Eclipse-Tips in case you'd like to give it a try

Reply:
Got it -- I haven't used that before, but it's handy.